ee/clickhouse/queries/paths/path_event_query.py

```diff
@@ -54,10 +54,13 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
             if self._should_query_url()
             else "if(0, '', "
         )
-        event_conditional += f"{self.EVENT_TABLE_ALIAS}.event)) AS path_item"
+        event_conditional += f"{self.EVENT_TABLE_ALIAS}.event)) AS path_item_ungrouped"
 
         _fields.append(event_conditional)
 
+        _fields.append("multiMatchAnyIndex(path_item_ungrouped, %(regex_groupings)s) AS group_index")
+        _fields.append("if(group_index > 0, %(groupings)s[group_index], path_item_ungrouped) AS path_item")
```

Comment:
Shouldn't we only apply these groupings to URLs?

Reply:
Oh, I was thinking there's nothing really stopping us from wildcarding custom events as well? Specially for compressing paths like: `insight loaded -> insight viewed -> insight analyzed`, we could have `insight *`.